source/common/ratelimit/ratelimit_impl.h

```diff
@@ -72,7 +78,7 @@ class NullClientImpl : public Client {
   // RateLimit::Client
   void cancel() override {}
   void limit(RequestCallbacks& callbacks, const std::string&, const std::vector<Descriptor>&,
-             const Tracing::TransportContext&) override {
```

Comment:
Nothing else in this repo I think now, but we have some internal filters that need to be updated before we can fully remove it.